.github/label-actions.yml

```diff
@@ -0,0 +1,22 @@
+# actions for Needs Logs label
+Needs Logs:
+  comment: "We need more info to debug your particular issue. If you could attach your logs to the issue, it would help us fix the issue much faster.
+
+1. Set the default logging level to `Debug` and enable logging to a file. To do this ensure the following entries are in your `host.json`. ([instructions](https://learn.microsoft.com/azure/azure-functions/configure-monitoring?tabs=v2#configure-log-levels))
+
+```json
+{
+  "logging": {
+      "fileLoggingMode": "debugOnly",
```

Comment:
"fileLoggingMode" needs to be set to "always" so that logs are generated when running locally (as well as in Portal).

Reply:
yes, I was also able too verify it after changing it to always.

Reply:
It's weird because I was seeing the logs in the file even with `debugOnly` set...

Anyways I'll update since that's what the docs seem to indicate - maybe I have some other local config messing it up.